docs/user_guide/queue_config.md

```diff
@@ -262,9 +272,10 @@ The star "*" is the wildcard character and matches all users or groups.
 Duplicate entries in the lists are ignored and do not cause a parsing error.
 Specifying a star beside other list elements is not allowed.
 
-_maxapplications_ is an unsigned integer value, larger than 1, which allows you to limit the number of running applications for the configured user or group.
+_maxapplications_ is an integer value, larger than 1, which allows you to limit the number of running applications for the configured user or group.
```

Comment:
unsigned was added on purpose as the parameter is defined as an `uint64` in the code. If you set a -1 an error unmarshalling the json will occur while a 0 should return a validation error. Probably better to keep the unsigned in the doc.

Reply:
I see, the reason I updated here is @pbacsko commented https://github.com/apache/yunikorn-core/pull/429#discussion_r961046263
If you think we should keep unsigned type, i will revert and update PR 429 as well.

Reply:
yes please do

Reply:
reverted.